src/sidebar/services/annotations.ts

```diff
@@ -53,12 +54,11 @@ export class AnnotationsService {
 
   /**
    * Extend new annotation objects with defaults and permissions.
-   *
-   * @param {Omit<AnnotationData, '$tag'>} annotationData
-   * @param {Date} now
-   * @return {Annotation}
    */
-  _initialize(annotationData, now = new Date()) {
+  private _initialize(
+    annotationData: Omit<AnnotationData, '$tag'>,
+    now: Date
```

Comment:
The default value on this argument was making code coverage drop.

Since it's a private method, there's no way to test both branches from a public API point of view, so I have removed the default value.

Reply:
I would be fine with just adding an `/* istanbul ignore next */` comment for this purpose, assuming the default value is useful.

Reply:
I'm not sure it makes sense to have a default value for a private method.

The public methods which end up calling it are always passing something, because their own logic requires them to conditionally determine the value.